src/gt4py/next/iterator/embedded.py

```diff
@@ -680,7 +680,18 @@ def _get_axes(
         assert all(first == _get_axes(f) for f in field_or_tuple)
         return first
     else:
-        return field_or_tuple.__gt_dims__
+        return field_or_tuple.__gt_domain__.dims
+
+
+def _get_domain(
```

Comment:
Just a question: if there is now a `_get_domain(foo)`, shouldn't then `_get_axes(foo)` be implemented as an alias to   `_get_domain(foo).ndims`?